backend/entityservice/views/run/description.py

```diff
@@ -1,5 +1,7 @@
+import psycopg2
 from flask import request
 from structlog import get_logger
+from retrying import retry
```

Comment:
retrying is unmaintained for more than a year now. https://github.com/rholder/retrying/issues/79
better: use tenacity https://github.com/jd/tenacity

Reply:
Ta didn't know that - have switched